vermouth/processors/locate_charge_dummies.py

```diff
@@ -193,7 +192,6 @@ def locate_all_dummies(molecule, attribute_tag=DEFAULT_DUMMY_ATTRIBUTE):
         dummy_keys = [pair[1] for pair in dummy_pairs]
         locate_dummy(molecule, anchor_key, dummy_keys, attribute_tag)
 
-
 class LocateChargeDummies(Processor):
     def __init__(self, attribute_tag=DEFAULT_DUMMY_ATTRIBUTE):
         super().__init__()
```

Comment:
This processor has no documentation on what it actually does apart form the technical aspect. This makes it impossible to review and also is not very sustainable as a package processor. I guess this is needed for the polarizeable side chains? @pckroon  can you provide some context.

Reply:
Indeed, it deals with polarizable side chains.

Reply:
@fgrunewald Completely agree. The first order of business should be tests though, so this fix wouldn't have been needed in the first place. Not this PR or release though.